src/web/src/pages/CoursePage.vue

```diff
@@ -75,6 +76,15 @@ export default {
       treeData.name = this.courseName;
       return treeData;
     },
+    getCredits() {
+      var credits;
+      if (this.courseObj.min_credits != this.courseObj.min_credits){
```

Comment:
I think this is a typo, left hand side should probably be `this.courseObj.max_credits`?